examples/common/entry/entry_sdl.cpp

```diff
@@ -217,22 +217,6 @@ namespace entry
 			{
 				_eventQueue.postAxisEvent(_handle, _gamepad, _axis, _value);
 
-				if (Key::None != s_axisDpad[_axis].first)
-				{
-					if (_value == 0)
-					{
-						_eventQueue.postKeyEvent(_handle, s_axisDpad[_axis].first,  0, false);
-						_eventQueue.postKeyEvent(_handle, s_axisDpad[_axis].second, 0, false);
-					}
-					else
-					{
-						_eventQueue.postKeyEvent(_handle
-								, 0 > _value ? s_axisDpad[_axis].first : s_axisDpad[_axis].second
-								, 0
-								, true
-								);
-					}
-				}
```

Comment:
Removing this code above stops the dpad buttons from being pressed when either stick is moved.